connectproto/proposervm/service.proto

```diff
@@ -20,3 +23,13 @@ message GetProposedHeightReply {
   // right now.
   uint64 height = 1;
 }
+
+// Request to get the current epoch.
+message GetCurrentEpochRequest {}
+
+// Response containing the current epoch.
+message GetCurrentEpochReply {
+  uint64 p_chain_height = 1;
```

Comment:
Could we document these fields as well since these proto files are basically the docs for the connect apis?